modules/misunderstood/src/views/full/util.ts

```diff
@@ -40,3 +40,16 @@ export const RESOLUTION = {
   [RESOLUTION_TYPE.qna]: lang.tr('module.misunderstood.qna'),
   [RESOLUTION_TYPE.intent]: lang.tr('module.misunderstood.intent')
 }
+
+export const groupEventsByUtterance = events => {
```

Comment:
_.groupBy does pretty much the exact same thing

Reply:
See comment I added: https://github.com/botpress/botpress/pull/4616/commits/f2c9ab361fc99c199baf5d5fd1db236d85eb0ec1
_.groupBy does not remember insertion order for keys